docs/en-US/component/page-header.md

```diff
@@ -98,27 +98,29 @@ The component is consisted of these parts
 </template>
 ```
 
-## Attributes
-
-| Name    | Description    | Type                  | Accepted Values | Default |
-| ------- | -------------- | --------------------- | --------------- | ------- |
-| icon    | icon component | `string \| Component` | —               | Back    |
-| title   | main title     | string                | —               | Back    |
-| content | content        | string                | —               | —       |
-
-## Events
-
-| Name | Description                         | Parameters |
-| ---- | ----------------------------------- | ---------- |
-| back | triggers when right side is clicked | —          |
-
-## Slots
-
-| Name       | Description        |
-| ---------- | ------------------ |
-| icon       | custom icon        |
-| title      | title content      |
-| content    | content            |
-| extra      | extra              |
-| breadcrumb | breadcrumb content |
-| default    | main content       |
+## API
+
+### Attributes
+
+| Name    | Description                   | Type                     | Default |
+| ------- | ----------------------------- | ------------------------ | ------- |
+| icon    | icon component of page header | ^[string] / ^[Component] | Back    |
+| title   | main title of page header     | ^[string]                | Back    |
```

Comment:
title doesn't seem to have a default value.

Reply:
Although the default value is not specified in prop, the default value is specified in the template.
Do I need to change `Back` to `''`? 
![image](https://user-images.githubusercontent.com/69044080/235332892-d0786e98-5a5e-4255-bb3a-967c7a2fd952.png)

Reply:
Yes, the default value is subject to the declaration in props, and the default value of the template can be considered in the description.

Reply:
updated